app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -35,14 +35,15 @@ def custom_index_relation(collection)
 
     def_param_group :repo do
       param :url, String, :desc => N_("repository source url")
+      param :required_tags, Array, :desc => N_("comma-separated list of tags used to identify whether the repository should be disabled on a client with a nonmatching release version")
```

Comment:
@jturel what do you think of the wording here?

Also unsure about the accuracy of "comma-separated list."  The browser sends it as an actual JSON array, yet `:docker_tags_whitelist` also says "comma-separated list," yet is an `Array`.  Is that because Hammer turns a comma-separated list into the correct format when it makes the API request?  Should it just say "list" here?

Reply:
Yeah, I think 'comma-separated' is a bit redundant when we're talking about an API field that is specified as an array!